ms2query/create_library_files.py

```diff
@@ -1,59 +1,58 @@
 from typing import List, Dict, Union
-import pandas as pd
 import os
+import pandas as pd
 from tqdm import tqdm
 from matchms.Spectrum import Spectrum
 from gensim.models import Word2Vec
 from ms2deepscore import MS2DeepScore
 from ms2deepscore.models import load_model as load_ms2ds_model
 from spec2vec.vector_operations import calc_vector
+from ms2query.utils import load_pickled_file
 from ms2query.create_sqlite_database import make_sqlfile_wrapper
 from ms2query.spectrum_processing import minimal_processing_multiple_spectra, \
     create_spectrum_documents
-from ms2query.app_helpers import load_pickled_file
 
 
 class CreateFilesForLibrary:
```

Comment:
The name made me more think of a function (which would also have been a viable solution to create the library files). If we use a class for it, maybe we can name it something like `LibraryFilesCreator`? "CreateFilesForLibrary" doesn't make clear when those files are created. Here the user actually has to call additional methods first before something is actually created.

Reply:
I changed the name to `LibraryFilesCreator`, feel free to pick/suggest another name!